composer.json

```diff
@@ -6,6 +6,13 @@
     "type": "project",
     "require": {
         "php": "^7.1.3",
+        "ext-bcmath": "*",
+        "ext-ctype": "*",
+        "ext-curl": "*",
+        "ext-json": "*",
+        "ext-mbstring": "*",
+        "ext-openssl": "*",
+        "ext-redis": "*",
```

Comment:
Please remove this. It will conflict with the default redis library pixelfed uses.

```suggestion
```